leetcode/client/auth.py

```diff
@@ -25,7 +25,12 @@
 class Auth(object):
     def __init__(self):
         self.cookies = None
-        self.get_cookies_from_chrome()
+        if os.path.exists(HOME.joinpath('.mozilla', 'firefox')):
+            self.get_cookies_from_firefox()
+        elif os.path.exists(HOME.joinpath('.config', 'google-chrome')): 
```

Comment:
I'd rather make brower configurable instead of using check like this. At least I don't have this chrome path on my mac system.

Reply:
That makes perfect sense. It was entirely just a duct tape and paperclips hack for my specific use.